packages/markdown-transform/lib/transform.js

```diff
@@ -26,6 +26,9 @@ const SlateTransformer = require('@accordproject/markdown-slate').SlateTransform
 const HtmlTransformer = require('@accordproject/markdown-html').HtmlTransformer;
 const PdfTransformer = require('@accordproject/markdown-pdf').PdfTransformer;
 const DocxTransformer = require('@accordproject/markdown-docx').DocxTransformer;
+const OoxmlTransformer = require('@accordproject/markdown-docx').OoxmlTransformer;
+const CiceroMarkToOOXMLTransfomer = require('@accordproject/markdown-docx').CiceroMarkToOOXMLTransfomer;
```

Comment:
Please follow the same naming convention as the other transformers. The name of the class is the transformation, while the methods indicate what is being transformed.